js/script.js

```diff
@@ -1,5 +1,42 @@
-function changeColor(){
-    document.body.className = 
-    (document.body.className === "bodyStyleLightBlue") ? 
-    "bodyStyleYellow" : "bodyStyleLightBlue";
-}
\ No newline at end of file
+let carsForTable = [];
+
+function convertToTableData(car) {
+    let carForTable = [];
+
+    carForTable.push(car.maker);
+    carForTable.push(car.model);
+    carForTable.push(car.year);
+    carForTable.push(car.color);
+    carForTable.push(car.price.price + " " + car.price.currency);
+
+    return carForTable;
+}
+
+
+
+$(document).ready(function() {
+
+    $.getJSON("http://localhost:8080/v1/car/list/", function(data) {
+    
+        for(const car of data) {
+            carsForTable.push(convertToTableData(car));
+        }
+
+        $('#carsTable').DataTable({
+            data : carsForTable,
+            columns : [
+                { title: "Maker" },
+                { title: "Model" },
+                { title: "Year" },
+                { title: "Color" },
+                { title: "Price" }
+            ]
+        });
+
+        let td=$('#carsTable td');
```

Comment:
nice try but it should be another best practice way using column properties at lines 27-32